### ISSUE TITLE
Tutorial feedback

### ISSUE THREAD

**Issue by kleshchevand** [tsunamayo/Starship-EVO#4299]
### Short Description of the issue:

1. Tutorial: Linking Tool
I spent dozen of minutes trying to find linking tool. Tutorial asked me to retrieve one from item stand, but it did not point out the item stand (there was a confusing pointer below the deck, but nothing on item stand), item stand was empty and item stand looks not like a stand but like a projector. I did figure to open Tab and drag linking tool eventually.

2. Tutorial reactor placement
I'm not sure if I missed something in tooltips, but it took me couple minutes to figure out where/how the reactor was supposed to be placed. I knew that I can stretch armor blocks, but reactor isn't armor so it didn't even cross my mind that reactors can be stretched too. And looking by steam forum discussion, I'm not alone with that issue.

3. Tutorial: repair
Using space ship I repaired two of the highlighted modules, but when I went for third, a hostile ship shot me down while I was trying to hide in the hangar. It also damaged the station. Eventually I repaired last module manually, and repaired repair ship. But for some reason I could not sit down into the repair ship, so I couldn't 'dock' to finish the tutorial. I can open and close cockpit, but any attempt to sit down does not work (might be because the ship was on it's side). Quitting and loading the game at that stage behaves weird (I end up stack in a wireframe of a station).

### Environment
- Starship EVO version: 21w47c
- Graphics Card: RTX 2060

[Player-prev.log](https://github.com/tsunamayo/Starship-EVO/files/7617320/Player-prev.log)

**Comment by tsunamayo** [comment]
@kleshchevand Hello, many thanks for the feedback. I will fix these asap and keep you posted!
Dont hesitate to add anything - it is really rare for me to get feedback from newcomers.
Cheers
Francois

**Comment by tsunamayo** [comment]
@kleshchevand
For 1) I am not sure what happen as I cant reproduce. But I now ask the player to take the Link Tool from the codex so this issue will be gone.
2) I added some messages in case the player get stuck at this step.
3) I added a safe zone of 4km around stations. Police Patrol ship will soon enforce it.
The quit to title and loading back issue is fixed.
Also the Mission Successful message now displays long enough.

Available for 21w48a. Thanks!
